admin/services/redis-cache.service.js

```diff
@@ -32,7 +32,7 @@ redisCacheService.get = async key => {
   redisConnect()
   try {
     const cacheEntry = await redis.get(key)
-    if (cacheEntry === null) return Promise.resolve(null)
+    if (cacheEntry === null) return null
```

Comment:
Why prefer null over undefined?  Generally, I would prefer to not use null in services.   And on that subject, shouldn't this service be in the data-access layer?

Reply:
this is a straight copy from the `tslib` version. will switch to undefined.

regards placement of the service - i agree, its a generic data service.  will be interesting to see how vscode handles the move 😄

Reply:
hang on a minute - i think there was a reason i used null, because `ioredis` returns null.  i can obviously return undefined, but the null check was necessary due to what the library returned.

will debug it.

Reply:
now returning `undefined` across all implementations.